scripts/setup/nrfconnect/get_zap.py

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env python3
 
+#!/usr/bin/env python3
```

Comment:
It's better to keep the shebang line as the first one. I'm not sure if all shells can handle a blank line.